lib/python/rosie/vc.py

```diff
@@ -233,12 +246,55 @@ def create(self, info_config, from_id=None, prefix=None,
         Return the SuiteId of the suite on success.
 
         """
+
         for key in ["owner", "project", "title"]:
             if not info_config.get([key], no_ignore=True):
-                raise SuiteInfoFieldError(key)
+                info_config_new = info_config
+                report_error = True
+                if mode_cli:
```

Comment:
I would have thought that these sort of checking should be done by `RosieVCClient.generate_info_config` instead of `RosieVCClient.create`?